docs/content/api.json

```diff
@@ -142381,7 +142381,7 @@
 	"readme": [
 		{
 			"kind": "text",
-			"text": "![DXOS](./docs/docs/assets/images/github-repo-banner.png)\n\n# DXOS\n\n[![MIT License](https://img.shields.io/npm/l/@dxos/client)](https://github.com/dxos/dxos/blob/main/LICENSE)\n[![CI](https://img.shields.io/circleci/build/gh/dxos/dxos/main?style=flat&token=dfa4d4d2d6b8a794bcb5f5772096beb2006cdfa8)](https://circleci.com/gh/dxos/dxos)\n![Netlify Status](https://img.shields.io/netlify/847d26d2-ec3b-4f2e-ac3b-caee7fe6e7bc)\n[![codecov](https://img.shields.io/codecov/c/github/dxos/dxos?token=CLO9PHX2PQ)](https://codecov.io/gh/dxos/dxos)\n![Node](https://img.shields.io/node/v/@dxos/client)\n[![NPM](https://img.shields.io/npm/v/@dxos/client/latest)](https://www.npmjs.com/package/@dxos/client)\n[![Discord](https://img.shields.io/discord/837138313172353095?label=discord)](https://discord.com/channels/837138313172353095)\n\nA decentralized alternative to the commercial cloud for privacy preserving software. Learn more about the [the mission](https://docs.dxos.org/guide/why)\n\n## Demo\n\nShown below are two sessions of an application using ECHO to replicate state peer-to-peer in real-time.\n\n![ECHO](https://user-images.githubusercontent.com/3523355/158708286-f9a8c5f1-83ed-4bac-ab9e-65ddb6861fe3.gif)\n\n## Documentation\n\n- [⚡️ Quick Start](https://docs.dxos.org/guide/getting-started.html)\n- [📖 Developer Guide](https://docs.dxos.org/guide)\n- [📚 API Reference](https://docs.dxos.org/api)\n- [🔧 Repository Guide](https://github.com/dxos/dxos/tree/main/REPOSITORY_GUIDE.md)\n\n## DXOS Resources\n\n- [Website](https://dxos.org)\n- [Developer Documentation](https://docs.dxos.org)\n- Talk to us on [Discord](https://discord.gg/eXVfryv3sW)\n\n## Contributions\n\nYour ideas, issues, and code are most welcome. Please take a look at our [community code of conduct](https://github.com/dxos/dxos/blob/main/CODE_OF_CONDUCT.md), the [issue guide](https://github.com/dxos/dxos/blob/main/CONTRIBUTING.md#submitting-issues), and the [PR contribution guide](https://github.com/dxos/dxos/blob/main/CONTRIBUTING.md#submitting-prs). To learn about how to set up for development and contribution to DXOS, see the [Repository Guide](https://github.com/dxos/dxos/tree/main/REPOSITORY_GUIDE.md)\n\nLicense: [MIT](./LICENSE) Copyright 2022 © DXOS"
```

Comment:
Sorry for the gigantic diff, the only change is from "docs/docs" to "docs/comment" like the other ones in this PR.